src/include/nanobench.h

```diff
@@ -1266,11 +1333,23 @@ void doNotOptimizeAway(T const& val) {
 #    endif
 #    if ANKERL_NANOBENCH(PERF_COUNTERS)
 #        include <map> // map
-
-#        include <linux/perf_event.h>
-#        include <sys/ioctl.h>
-#        include <sys/syscall.h>
 #        include <unistd.h>
+#        if defined(__linux__)
+#           include <linux/perf_event.h>
+#           include <sys/ioctl.h>
+#           include <sys/syscall.h>
+#        endif
+#        if defined(RISCV_PERF_COUNTERS)
```

Comment:
This is newlib-specfiic issue, not required if we're compiling for Linux using Linux toolchain. check macro for newlib - don't use RISCV_PERF_COUNTERS.

Reply:
e.g. #ifndef __GLIBC__